packages/@sanity/default-layout/package.json

```diff
@@ -35,23 +35,23 @@
     "@sanity/client": "2.19.0",
     "@sanity/generate-help-url": "2.18.0",
     "@sanity/ui": "^0.36.8",
+    "@sanity/icons": "^1.1.7",
     "@sanity/util": "2.20.0",
-    "classnames": "^2.2.5",
-    "is-hotkey": "^0.1.6",
     "lodash": "^4.17.15",
-    "react-click-outside": "^3.0.0",
     "react-props-stream": "^1.0.0",
+    "react-rx": "^1.0.0-beta.6",
     "rxjs": "^6.5.3"
   },
   "devDependencies": {
+    "@sanity/state-router": "2.20.0",
```

Comment:
I think we can remove this, by changing imports to `import x from '@sanity/base/router'`